app/storage.tsx

```diff
@@ -61,6 +71,74 @@ class Storage {
     }, null);
   }
 
+  // Insert expansion pack into the database
+  parseExpansionPack(expansionData) {
+    this.db.transaction((tx) => {
+      tx.executeSql(
+        'INSERT INTO expansionPackTable (pack_name, description, wtw, organisation) VALUES (?,?,?,?)',
+        [
+          expansionData.meta.pack_name,
+          expansionData.meta.description,
+          expansionData.meta.w3w,
+          expansionData.meta.organisation
+        ]
+      );
+      tx.executeSql(
+        'SELECT id FROM expansionPackTable WHERE pack_name=?',
+        ['Boyd Orr'],
```

Comment:
Does this not mean only `'Boyd Orr'` expansion pack ID is used?

Reply:
ah, yes this is an error! Thanks :)